README.md

```diff
@@ -66,82 +75,84 @@ The `.deb` that Tauri builds will automatically do this for us for actual toolki
 
 5. Close your current terminal and open a new one.
 
-6. Install rust.
+6. Install Rust:
 
     ```bash
-    $ curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
+    curl --proto '=https' --tlsv1.2 https://sh.rustup.rs -sSf | sh
     ```
 
-7. Install volta or node. Volta now no longer support Apple silicon chip series (M series) therefore we can use alternative like node instead
+7. Install Volta or Node. Volta now no longer support Apple silicon chip series (M series). Therefore we can use an alternative like Node instead:
 
     ### for Apple silicon series user
 
     ```bash
-    $ sudo apt install nodejs npm
+    sudo apt install nodejs npm
     ```
 
-    ### for window machine with Intel/AMD
+    ### for Windows machine with Intel/AMD
 
     ```bash
-    $ curl https://get.volta.sh | bash
+    curl https://get.volta.sh | bash
     ```
 
 8. Close your current terminal and open a new one.
 
-9. Install node.
+9. Install Node:
 
     ### for Apple silicon series user
 
     ```bash
-    $ npm install node
+    npm install node
     ```
 
-    ### for window machine with Intel/AMD
+    ### for Windows machine with Intel/AMD
 
     ```bash
-    $ volta install node
+    volta install node
     ```
 
-10. Install yarn.
+10. Install Yarn:
 
     ### for Apple silicon series user
 
     ```bash
-    $ npm install yarn
+    npm install yarn
     ```
 
-    ### for window machine with Intel/AMD
+    ### for Windows machine with Intel/AMD
```

Comment:
for Windows machines with an Intel/AMD chipset

Reply:
implemented in commit https://github.com/Hardhat-Enterprises/Deakin-Detonator-Toolkit/pull/489/commits/eeb11e0cfe614a7a9ba65bd02e359ff023c7670c